cfme/utils/smem_memory_monitor.py

```diff
@@ -1,6 +1,5 @@
 """Monitor Memory on a CFME/Miq appliance and builds report&graphs displaying usage per process."""
-import matplotlib.dates as mdates
-import matplotlib.pyplot as plt
```

Comment:
Instead of moving the imports into each method, if we're simply trying to avoid choosing the tk backend, why can't we do this:

```python
import matplotlib
matplotlib.use('TheBackEndWeWant')
import matplotlib.dates as mdates
import matplotlib.pyplot as plt
```

See example of that here: https://matplotlib.org/faq/usage_faq.html

Reply:
yes, but then we get linting issues ^^

Reply:
IMO I think `# noqa` on that `.use` line would be worth it here :) Importing once at the top is simple. If we explicitly define the backend up top we will be able to guarantee no one hits the tkinter dependency issue.